catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -68,10 +77,31 @@
 
     private boolean enableDelete = true;
 
+    private boolean initialized = false;
+
+    private ScheduledExecutorService executor;
+
+    private int taskWaitTimeSeconds = 30;
+
+    private int refreshInterval = 30;
+
+    private Future scheduledTask;
+
     public RefreshRegistryEntries() {
 
     }
 
+    public void init() {
+        scheduledTask = executor.scheduleWithFixedDelay(() -> {
+            try {
+                refreshRegistryEntries();
+            } catch (FederationAdminException e) {
+                LOGGER.debug("Problem refreshing registry entries.", e);
+            }
+        }, 10, refreshInterval, TimeUnit.SECONDS);
```

Comment:
What will happen if it takes more than the refresh interval to update the registry entries?